lib/matplotlib/tests/test_figure.py

```diff
@@ -557,6 +557,33 @@ def test_savefig_pixel_ratio(backend):
     assert ratio1 == ratio2
 
 
+@image_comparison(['transparent_background'],
+                    extensions=['png'], savefig_kwarg={'transparent': True},
+                    remove_text=True)
+def test_savefig_transparent():
```

Comment:
@anntzer one reason I did not implement the complete empty plot test mentioned in the previous PR was that that test would pass even if there is nothing being done while plotting?

Reply:
Yes, but it's extremely unlikely that we'd ever introduce a bug such that `ax.spines[:].set_visible(False); ax.set(xticks=[], yticks=[])` would actually disable all drawing.
If you want to feel extra safe I would still do a check_figures_equal where the reference figure has all axes *manually* made transparent with set_facecolor("none") calls.

Reply:
Ah okay. Yeah, I'll do that then. Thanks.

Reply:
Needed a quick help. Since this is related to a savefig parameter, I'll need to pass that to the savefig function inside the decorator. From what I can see, that's not possible right now with `check_figures_equal`? Can we add that? I guess it'll be useful in other cases too..

Reply:
@anntzer any suggestions on this?

Reply:
Something like
```python
@mpl.rc_context({"savefig.transparent": True})
@check_figures_equal(extensions=["png"])
def test_savefig_transparent(fig_test, fig_ref):
    # create two transparent subfigures with corresponding transparent inset
    # axes. the entire background of the image should be transparent.
    gs1 = fig_test.add_gridspec(3, 3, left=0.05, wspace=0.05)
    f1 = fig_test.add_subfigure(gs1[:, :])
    f2 = f1.add_subfigure(gs1[0, 0])

    ax12 = f2.add_subplot(gs1[:, :])

    ax1 = f1.add_subplot(gs1[:-1, :])
    iax1 = ax1.inset_axes([.1, .2, .3, .4])
    iax2 = iax1.inset_axes([.1, .2, .3, .4])

    ax2 = fig_test.add_subplot(gs1[-1, :-1])
    ax3 = fig_test.add_subplot(gs1[-1, -1])

    for ax in [ax12, ax1, iax1, iax2, ax2, ax3]:
        ax.set(xticks=[], yticks=[])
        ax.spines[:].set_visible(False)

    # and draw nothing on fig_ref.
```
?

Reply:
Did not know about `mpl.rc_context`. This should work, thanks!